### PR TITLE
[infra] Introduce compiler ARM cross build script

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce compiler ARM cross build script using Docker.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

File: infra/scripts/docker_build_cross_arm_compiler_debug.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  DOCKER_ENV_VARS+=" -e EXTERNAL_VOLUME=/externals"
+else
+  echo "It will use default external path"
+fi
+
+DOCKER_ENV_VARS+=" -e BUILD_TYPE=Debug"
+DOCKER_ENV_VARS+=" -e BUILD_JOBS=4"
+
+# docker image name
+# - for bionic, use DOCKER_IMAGE_NAME="nnfw/one-devtools:bionic"
```
Comment (seanshpark) [inline infra/scripts/docker_build_cross_arm_compiler_debug.sh:28]: @hseok-oh , is this comment OK? or is there for `nncc` ?

**Comment by seanshpark** [comment]
for #9838, from  #9839

File: infra/scripts/docker_build_cross_arm_compiler_debug.sh (hunk 1 above)
Comment (hseok-oh) [reply infra/scripts/docker_build_cross_arm_compiler_debug.sh:28]: It's OK. Compiler and runtime share docker image.

**Review by hseok-oh — APPROVED** [review]
LGTM

**Pull request merged**